src/twisted/internet/test/connectionmixins.py

```diff
@@ -147,9 +151,12 @@ def runProtocolsWithReactor(
 
     @param endpointCreator: An instance of L{EndpointCreator}.
 
+    @param reactorSetUp: Callable called with the reactor as argument, after the reactor is created.
```

Comment:
I think it's cleaner to pass a subclass of reactorBuilder with a custom reactorFactory

Reply:
Hmm, good point, `reactorBuilder` effectively already has this functionality?